projects/vilbert/configs/visual7w/defaults.yaml

```diff
@@ -0,0 +1,53 @@
+model_config:
+  vilbert:
+    training_head_type: visual7w
+    num_labels: 1
+    losses:
+    - type: cross_entropy
+
+dataset_config:
+  hateful_memes:
```

Comment:
This should be `visual7w` or your dataset name. This means your experiments are not using bert tokenizer. Maybe this is also a reason for lower scores.

Reply:
If this is not required remove this.

Reply:
Done